README.md

```diff
@@ -71,11 +71,28 @@ The returned event payload is the same as the input.
 `SendRichMessageFailed`
 ```json
 {
-  "inputMessage": { ... },
+  "inputMessage": {
+  },
   "error": "..."
 }
 ```
 
+
+### GetReactionMessageInfo
+This command would retrieve the message text of reaction sent by a user. 
+
+```
+{                   "count":  50 , //default value is 100 mandatory
```

Comment:
Not sure yet what this is doing. What `count` means?

Reply:
The count of reactions  in list sorted via timestamp. Hope this clarifies.